crates/db_schema/src/source/comment.rs

```diff
@@ -1,11 +1,10 @@
 use crate::newtypes::{CommentId, DbUrl, LanguageId, LtreeDef, PersonId, PostId};
+#[cfg(feature = "full")]
+use crate::schema::{comment, comment_like, comment_saved};
```

Comment:
This isnt so nice, it doesnt keep imports with feature declaration in a separate block.

Reply:
Not much we can do about that, oh well.